app/src/main/java/edu/akelael/comics/model/ComicListModel.java

```diff
@@ -0,0 +1,48 @@
+package edu.akelael.comics.model;
+
+import java.util.HashMap;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class ComicListModel implements IComicListModel {
+    Server server;
+    Retrofit retrofit;
+    HashMap<String, String> mMap;
```

Comment:
I believe these three attributes doesn't need to be package private. Their visibility can be made more restrictive probably. Also, IMO their names doesn't tell what they are for. mMap is map, that's something that can be seen in the declaration, but what it really improves readability is to explain what is used for. For example, I would name it something like "apiRequestContent", as it includes the timestamp, api key and hash for making a request to the Marvel API.

On the other hand, the hungarian notation (that includes adding an m before the name of a field) was used when we didn't have syntax highlights in our IDEs and it helped the programmers to realize that was a field. Old projects can still use it as a part of their legacy (code), but now is completely useless, and it worsens legibility.